testing/functional_tests/t_template.py

```diff
@@ -56,7 +56,8 @@
 
 import logging
 import os
-from t_utils import remove_tmp_dir, create_tmp_dir, __data_testing_dir__, __tmp_dir__
+from testing.functional_tests.t_utils import create_tmp_dir, __data_testing_dir__, __tmp_dir__
+from testing.common_testing_util import remove_tmp_dir
```

Comment:
Most updates and below are similar in nature: instead of importing specific t_utils.py variables, these are centralized. 

Next test update PR will likely address the __data_test_dir__ and __tmp_dir__ as most underscore prefixed variables are not meant to be past around by naming conventions.